auth/request-uri.md

```diff
@@ -29,5 +29,5 @@
     relay = { protocol: “iridium”, data: “” }
 
 ```
-uri = “wc:7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=iridium&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303”
+uri = “wc:auth-7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=iridium&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303”
```

Comment:
Can we change `iridium` to `irn` here?

Reply:
good catch, I think it should be changed